test/api/schemas/response/keyword/success.json

```
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string"
+            },
+            "id": {
+              "type": "string"
```

```suggestion
              "type": "string",
              "format": "uuid"
```

Does it work with UUID here? According to [json-schema](https://json-schema.org/draft/2020-12/json-schema-validation#section-7.3.5-2.10) we could validate this format instead of allowing any random string 👍